src/components/calendar/Calendar.vue

```diff
@@ -81,60 +87,81 @@
 			<div id="legend">
 				<!-- Dummy first node to leave space for header -->
 				<p></p>
-				<p v-for="(hour, i) in timeRange()">{{ hour }}</p>
+				<p v-for="(hour, i) in timeRangeDisplay()" :key="i">{{ hour }}</p>
 			</div>
-			<Day v-for="(day, i) in dayModels" :dayModel="day" :timeRangeLength="timeRangeLength" />
+			<div id="leftBgCol"></div>
+			<Day v-for="(day, i) in dayModels" :key="i" :dayModel="day" :timeRangeLength="timeRangeLength" />
+			<div id="rightBgCol"></div>
 		</div>
 	</div>
-	<Footer />
+	<!-- <Footer /> -->
```

Comment:
why?

Reply:
oops, it wasnt sticky on the bottom and the spacing around the edge annoyed me so i disabled it lol
might be worth its own issue

Reply:
#46